### PR TITLE
Use tags on features/scenarios to allow skipping tests

### PR DESCRIPTION
**Pull request by edwardsph**

### CODE REVIEW

**Review by kjetilk — APPROVED** [review]
One comment about ED vs TR, other than that, looks good.

**Review by kjetilk — CHANGES_REQUESTED** [review]
Eh, it was the intention to request a change :-)

File: application.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 subjects: /data/test-subjects.ttl
 sources:
   # Protocol spec & manifest
-  # Editor's draft (fully annotated)
-  - https://solidproject.org/TR/protocol
+  # Editor's draft (Version 0.9.0 - https://solidproject.org/TR/2021/protocol-20211217)
+  - https://solidproject.org/ED/protocol
```
Comment (edwardsph) [inline application.yaml:5]:
```suggestion
  # Ecosystem Proposal (Version 0.9.0 - https://solidproject.org/TR/2021/protocol-20211217)
  - https://solidproject.org/TR/protocol
```

File: protocol/solid-protocol-test-manifest.ttl [hunk 2]
```diff
@@ -5,8 +5,11 @@ prefix dcterms: <http://purl.org/dc/terms/>
 prefix td: <http://www.w3.org/2006/03/test-description#>
 prefix spec: <http://www.w3.org/ns/spec#>
 
-# Editor's draft (fully annotated)
-prefix sopr: <https://solidproject.org/TR/protocol#>
+# Editor's draft
+prefix sopr: <https://solidproject.org/ED/protocol#>
```
Comment (edwardsph) [inline protocol/solid-protocol-test-manifest.ttl:9]:
```suggestion
# Ecosystem Proposal
prefix sopr: <https://solidproject.org/TR/protocol#>
```

**Review by edwardsph — COMMENTED** [review]
Good to go?

**Pull request merged**